src/ameba/ast/visitors/implicit_return_visitor.cr

```diff
@@ -181,6 +181,14 @@ module Ameba::AST
       false
     end
 
+    def visit(node : Crystal::TypeOf) : Bool
+      report_implicit_return(node)
+
+      incr_stack { node.accept_children(self) }
+
+      false
+    end
+
```

Comment:
IIRC support for this was ruled out since `typeof` is used in specs because of its side-effects.

Reply:
I see, thanks for clarification ❤️